### ISSUE TITLE
App Bundle Module files can be only in pre-defined dirs

### ISSUE THREAD

**Issue by erikpowa** [xamarin/xamarin-android#2969]
of #2841 caught

`[BT : 0.8.0] error : Module files can be only in pre-defined directories, but found 'play-services-auth.properties'.`

Root of the `base.zip` got `.properties` files also custom folders like `kotlin` with files that are not yet handled.

**Comment by jonathanpeppers** [comment]
@erikpowa how are you adding these files to your Xamarin.Android project?

Are they `AndroidResource`, `AndroidAsset`, etc.?

**Comment by erikpowa** [comment]
@jonathanpeppers I don't. These are in `classes.jar` and being extracted during build to `$(IntermediateOutputPath)android\bin` or atleast where I got artifacts from are having those files, fast sample [link](https://maven.google.com/com/google/android/gms/play-services-auth/16.0.1/play-services-auth-16.0.1.aar). 
[Here](http://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-stdlib-common/1.3.30/kotlin-stdlib-common-1.3.30.jar) is another sample with bunch of `kotlin_metadata`.

**Comment by jonathanpeppers** [comment]
So you just have some `.aar` files with `<AndroidAarLibrary>`? (I'll try this)

**Comment by erikpowa** [comment]
@jonathanpeppers No, it comes from binding, so either `jar` or `__AndroidLibraryProjects__`. I just linked sample artifact that contains files that are extracted to the root of the `apk` / `base.zip`.
Hmm, this `[Xamarin.GooglePlayServices.Auth](https://www.nuget.org/packages/Xamarin.GooglePlayServices.Auth/71.1600.0-preview2)` has as well.

**Comment by jonathanpeppers** [comment]
I can repro it in our unit test with:
```diff
diff --git a/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/BundleToolTests.cs b/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/BundleToolTests.cs
index b83c1d07..1dad45af 100644
--- a/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/BundleToolTests.cs
+++ b/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/BundleToolTests.cs
@@ -19,7 +19,7 @@ namespace Xamarin.Android.Build.Tests
                [OneTimeSetUp]
                public void OneTimeSetUp ()
                {
-                       project = new XamarinAndroidApplicationProject {
+                       project = new XamarinFormsMapsApplicationProject {
                                IsRelease = true,
                        };
```

**Comment by muhaym** [comment]
![image](https://user-images.githubusercontent.com/10905479/60224183-00135600-989f-11e9-9bfb-a68767b61f27.png)
Around 40+% Size Reduction in AOT mode <3
